lib/cylc/remote.py

```diff
@@ -306,9 +308,8 @@ def execute(self, dry_run=False, forward_x11=False, abort_if=None,
             return False
 
         if abort_if is not None and abort_if in sys.argv:
-            sys.stderr.write(
-                "ERROR: option '%s' not available for remote run\n" % abort_if)
-            return True
+            sys.exit(
+                "ERROR: option '%s' not available for remote run" % abort_if)
```

Comment:
Previously, this would print out an error, and the command (e.g. `cylc cat log edit` will still exit 0. Now the command will exit 1 with the error message.